### PR TITLE
successively look for .data('path') up parents

### PR DESCRIPTION
**Pull request by rgbkrk**
Closes https://github.com/jupyter/notebook/issues/4865

This affects Notebook 6.x.

I'd prefer to refactor this, but I just want to see this working for the moment.

### CODE REVIEW

File: notebook/static/tree/js/kernellist.js [hunk 1]
```diff
@@ -85,7 +85,14 @@ define([
             .addClass('btn btn-warning btn-xs')
             .text(i18n._('Shutdown'))
             .click(function() {
-                var path = $(this).parent().parent().parent().data('path');
+                var parent = $(this).parent().parent().parent();
+                var path = parent.data('path'):
```
Comment (jasongrout) [inline notebook/static/tree/js/kernellist.js]: This line should end in a semicolon.

**Comment by jasongrout** [comment]
This fix works great for me once the last character in the line mentioned above is changed to a semicolon.

Thanks!

**Comment by jasongrout** [comment]
Thanks @rgbkrk! Love your branch name...

**Comment by rgbkrk** [comment]
> Love your branch name...

Glad you noticed!

File: notebook/static/tree/js/kernellist.js (hunk 1 above)
Comment (rgbkrk) [reply notebook/static/tree/js/kernellist.js]: Thanks for adding it in to fix the build! I can't believe I sent that. 😊

**Comment by afriedman412** [comment]
where is this file located?

(I'm running jupyter through anaconda 3, none of the paths i bring up through jupyter --paths have a notebook/ containing static/ or a static/ containing tree/)

**Pull request merged**